auditors/apparmor/fixtures/apparmor-disabled-overriden-old-label.yml

```diff
@@ -0,0 +1,14 @@
+# this is to test backwards compatibility with old unregistered annotations (kubernetes.io)
+apiVersion: v1
```

Comment:
tests container override labels for the old annotation, so we make sure it's all still working